pages/archive.tsx

```diff
@@ -156,7 +156,7 @@ const PopUp = ({ event, close }: { event: CyaneaEvent; close: () => void }) => {
           <div className={popUpStyle["date"]}>{formatEventDate(event)}</div>
           <div className={popUpStyle["location"]}>{event.location}</div>
           <div className={styles.tags}>
-            <span>Tags: </span>
+            <div>Tags:</div>
```

Comment:
Is there a reason this isn't a `<div><span>`?

Reply:
Not really, would you mind explaining the difference?

Reply:
Not really. Just changed it to a ```<div><span>```.

Reply:
I think @Aplet123 was trying to say that i should be `<div className=...><span>` rather than changing the inner `<div>` to a `<div>` + `<span>`, @stewartkwokca.

Reply:
Ah, maybe I'm confused but I don't think the span ever had any classes before? I just wanted to move it onto its own line to be consistent with everything else.

Reply:
Will work on cards a little more first